packages/vuetify/src/components/VFileInput/__tests__/VFileInput.spec.cy.tsx

```diff
@@ -136,4 +136,58 @@ describe('VFileInput', () => {
       .get('.v-file-input input')
       .should('have.attr', 'accept', 'image/*')
   })
+
+  it('should reset the underlying HTMLInput when model is controlled input', () => {
+    function TestWrapper () {
+      const files = ref<File[]>([])
+      const onReset = () => {
+        files.value = []
+      }
+      return (
+        <CenteredGrid width="400px">
+          <VFileInput modelValue={ files.value } />
+          <button onClick={ onReset }>Reset Model Value</button>
+        </CenteredGrid>
+      )
+    }
+
+    cy.mount(() => (
+      <TestWrapper />
+    ))
+      .get('.v-file-input input')
+      .should($res => {
+        const input = $res[0] as HTMLInputElement
+        expect(input.files).to.have.length(0)
+      })
+      .get('.v-file-input input')
+      // add file
+      .attachFile('text.txt')
+      .get('.v-file-input input')
```

Comment:
I think you should be able to remove all the `get`s after the first one. The context should still be the input element.